### ISSUE TITLE
Add timestamp parameter for 'Todo` data type and function to create 'Todo' with time

### ISSUE THREAD

**Issue by chshersh** [StasyaGera/todo-cli#13]
Use this data type from `time` package:

* https://hackage.haskell.org/package/time-1.9.1/docs/Data-Time-Clock.html#t:UTCTime

Also, I suggest to store `Maybe UTCTime` to be able to perform pure property-based tests without knowing something about `IO`.

**Comment by StasyaGera** [comment]
I have started doing this task and faced with a difficulty.
`UTCTime` is not a `hashable` instance, so I guess I have to implement it by myself.
After some googling I found [this](https://hackage.haskell.org/package/hashable-time-0.2.0.1/docs/Data-Hashable-Time.html). Should I import it or just copy some instances?

**Comment by StasyaGera** [comment]
I also think I should change `Deadline Int` to `Deadline UTCTime`.

**Comment by chshersh** [comment]
@StasyaGera I don't think you need `Hashable` instance for `UTCTime` and you can avoid implementing it. You can just translate `UTCTime` to some integer (unix timestamp) and hash that integer. Since you're implementing `instance Hashable Deadline` you have more options about what to do.
